packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -93,7 +101,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     return (
       <div ref='root'>
         { label && (
-          <label id={ id + '-label' } className='ms-Label' ref={ (dropdownLabel) => this._dropdownLabel = dropdownLabel } >{ label }</label>
+          <Label id={ id + '-label' } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
```

Comment:
It's was before me, I just added Label component. Probably for some needs it was done :) 
@dzearing, can I remove this without any side-effects?